assets/js/modules/analytics/components/dashboard/DashboardAllTrafficWidget/UserDimensionsPieChart.js

```diff
@@ -207,17 +207,16 @@ export default function UserDimensionsPieChart( { dimensionName, entityURL, sour
 	}
 
 	return (
-		<div className="googlesitekit-widget--analyticsAllTrafficV2__dimensions-chart">
+		<div className="googlesitekit-widget--analyticsAllTraffic__dimensions-chart">
 			<GoogleChart
-				chartID="user-dimensions-pie-chart"
```

Comment:
@benbowler the `chartID` is important for this chart and it should be unique. Please, use the `useInstanceId` hook again to make the `chartID` property unique on per instance basis. Plus, make sure that `GoogleChart.charts.get( 'user-dimensions-pie-chart' )` calls in the `onReady` callback use the same chart id as we pass to the `GoogleChart` component.

Reply:
Resolved.